### PR TITLE
Add option to not prepare __REF_DATA table to Prepare [VS-697]

### PR DESCRIPTION
**Pull request by rsasch**
Since we might need to run the GvsPrepareRangesCallset.wdl in the future just to help to get the callset stats, add an optional input to not create the __REF_DATA and __SAMPLES tables, since those won't be needed.

Test runs: 
 - [with](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/1ee2bc37-abd4-4185-a17d-e1f6b47a6914) `skip_ref_ranges_tables ` set to `true`:
 - [with](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/2849dfd1-fe41-4757-83f3-fc0f5451eaef) `skip_ref_ranges_tables ` set to `false`
 - [with](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/5e8c5c4d-6ae5-4117-a5ea-89a05d379cfd) `skip_ref_ranges_tables ` not set (default is false)
 - [GvsCallsetStatistics run](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/3e14b26f-125b-4742-b7a2-17cab2ac7f6f)
... after changes ... 
- [with](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/3560a8a6-71ef-46b4-be64-fd35ea5303c6) `only_output_vet_tables` set to `true`
 - [with](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/ac3a2126-c3f1-4241-aad5-55ece4178a56) `only_output_vet_tables` set to `false`
- [GvsCreateCohortFromSampleNames](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/35379d1b-b54a-4f5e-990a-9650c92d1ead) run (failed because of other issue)

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]
yay! 👍

**Review by koncheto-broad — APPROVED** [review]
LGTM

File: scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl [hunk 1]
```diff
@@ -98,14 +101,16 @@ task PrepareRangesCallsetTask {
           --query_project ~{query_project} \
           ~{sep=" " query_label_args} \
           --fq_sample_mapping_table ~{fq_sample_mapping_table} \
-          --ttl ~{temp_table_ttl_in_hours}
+          --ttl ~{temp_table_ttl_in_hours} \
+          ~{true="--only_output_vet_tables True" false='' only_output_vet_tables}
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl:105]: TOL this WDL syntax... 🙈

File: scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl [hunk 2]
```diff
@@ -264,7 +264,7 @@ task Add_AS_MAX_VQSLOD_ToVcf {
     File input_vcf
     String output_basename
 
-    String docker = "us.gcr.io/broad-dsde-methods/variantstore:2022-10-25-alpine"
+    String docker = "us.gcr.io/broad-dsde-methods/variantstore:2022-10-28-alpine"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl:267]:
```suggestion
    String docker = "us.gcr.io/broad-dsde-methods/variantstore:2022-11-07-v2-alpine"
```

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py [hunk 3]
```diff
@@ -72,13 +72,19 @@ def load_sample_names(sample_names_to_extract, fq_temp_table_dataset):
     return fq_sample_table
 
 
-def get_all_sample_ids(fq_destination_table_samples):
-    sql = f"select sample_id from `{fq_destination_table_samples}`"
+def get_all_sample_ids(fq_destination_table_samples, only_output_vet_tables, fq_sample_mapping_table):
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:75]: oops this was not doing the right thing before? 😬

**Review by mcovarr — APPROVED** [review]

File: scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl:105]: I just learned it, too — handy, right?!

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py (hunk 3 above)
Comment (rsasch) [reply scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:75]: 🙈

**Pull request merged**